numbers.js

```diff
@@ -1 +1,16 @@
 //This file contains Javascript functions that is operated on numbers.
+
+function fibo(range){
+    var f1=0
```

Comment:
As a java script programmer, Its a good habit to use let and const type of declaration.